lib/puppet/type/ec2_launchconfiguration.rb

```diff
@@ -52,9 +52,17 @@ def insync?(is)
     end
   end
 
+  newparam(:vpc) do
+    desc 'A hint for detecting ambiguously named groups liked default.'
```

Comment:
Seems like a typo (liked/like) but even corrected this could be a little more verbose.